pkg/kyc/component_test.go

```diff
@@ -420,28 +419,14 @@ func TestValidateUser(t *testing.T) {
 		mocks.keycloakClient.EXPECT().UpdateUser(accessToken, targetRealm, userID, gomock.Any()).Return(nil)
 		mocks.usersDB.EXPECT().StoreOrUpdateUserDetails(ctx, targetRealm, gomock.Any()).Return(nil)
 		mocks.usersDB.EXPECT().CreateCheck(ctx, targetRealm, userID, gomock.Any()).Return(nil)
-		mocks.eventsDB.EXPECT().ReportEvent(gomock.Any(), "VALIDATE_USER", "back-office", gomock.Any())
+		mocks.eventsReporter.EXPECT().ReportEvent(gomock.Any(), gomock.Any())
```

Comment:
Checking the event type can be useful when you need to maintain this code. (you know which event is expected)